lib/models/user.rb

```diff
@@ -4,4 +4,5 @@ class User
   include DataMapper::Resource
 
   property :id, Serial, key: true
-end
\ No newline at end of file
+  property :name, String, length: 50
```

Comment:
Strings by default have a max length of 255 characters. There's not really a good reason to limit it here. Just something to think about - not a blocker.